pkg/expapi/deep_copy.go

```diff
@@ -0,0 +1,16 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
hack/verify-boilerplate.sh is failing on Travis because of invalid boilerplate.
(There should be a line break before this)